.gitignore

```diff
@@ -46,6 +46,7 @@
 /platform/o.n.bootstrap/launcher/windows/*.dll
 /platform/o.n.bootstrap/launcher/windows/*.exe
 /platform/o.n.bootstrap/launcher/windows/*.res
+/java/java.lsp.server/vscode/*.vsix
```

Comment:
Why would be a `.vsix` file created in this directory?

Reply:
It was created by manually invoking `vsce package`, removing.